t/009-error-log-append.t

```diff
@@ -0,0 +1,211 @@
+use Test::Nginx::Socket::Lua;
+
+#worker_connections(1014);
+#master_process_enabled(1);
+#log_level('warn');
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 5) -2;
+
+#no_diff();
+no_long_string();
+#master_on();
+#workers(2);
+check_accum_error_log();
+run_tests();
+
+__DATA__
+
+=== TEST 0: Dummy: remove this test and the SKIP section from all tests below
```

Comment:
Should we start the test cases from `1`?

Reply:
we should indeed, but this first test case is a dummy, which will be removed once we can actually run the rest of the tests (see description). 

It is required otherwise running the file with no tests enabled produces a failure.